contrib/win32/win32compat/fileio.c

```diff
@@ -450,7 +450,7 @@ fileio_open(const char *path_utf8, int flags, mode_t mode)
 		cf_flags.dwFlagsAndAttributes, NULL);	
 
 	if (handle == INVALID_HANDLE_VALUE) {
-		errno = errno_from_Win32LastError();
```

Comment:
Instead modify errno_from_Win32LastError() to map the returned error code to ENOENT.